### DISCUSSION TITLE
mtex only able to detect a single phase when importing EBSD

### DISCUSSION THREAD

**Discussion by LITINGG** [mtex-toolbox/mtex#1257] [Ask Anything]
Hi guys !
i have imported a txt file into mtex, but it detect the file as single phase. hence the GUI selection only allow me to input details for one phase, so there is no grain boundary on the plot at all
[14_10 test.txt](https://github.com/mtex-toolbox/mtex/files/7340564/14_10.test.txt)

the figure should turn out as followed 
![image](https://user-images.githubusercontent.com/89003422/137193974-f0d2a5eb-2122-46c3-9077-13a2827bce99.png)

however it is shown as a single phase plot as shown below
![image](https://user-images.githubusercontent.com/89003422/137194136-4fbc9465-2596-423a-a8f6-580b0e9b7eac.png)

may i know if the issue lies with my txt file format or do i  have to add in extra code for the other phase, for mtex to recognize and plot the grain boundaries? 

Appreciate the help! thank you!

**Comment by zmichels** [comment]
Is one of the columns in your .txt file indicating phase?

**Reply by LITINGG** [reply]
yes! The 'I_state' column actually indicates the phase from 0 to 3 arrange in order
